seed/lib/superperms/orgs/models.py

```diff
@@ -42,6 +42,16 @@
     (STATUS_REJECTED, 'Rejected'),
 )
 
+# Measurement systems for user's display preference and initial choice at import time
+MEASUREMENT_US = 1
+MEASUREMENT_SI_GJ = 2  # cdn federal government reporting, portfolio manager
+MEASUREMENT_SI_KWH = 3 # cdn municipalities, industry working units
+
+MEASUREMENT_SYSTEMS = (
+    (MEASUREMENT_US, 'United States customary units - kBtu/sq ft'),
+    (MEASUREMENT_SI_GJ, 'Metric (SI) units - GJ/m2'),
+    (MEASUREMENT_SI_KWH, 'Metric (SI) units - kWh/m2'),
```

Comment:
This could definitely spiral out of control. What if we have a second level of configuration that provides specific units for various fields. For example, there would be another field for "units of EUI" with another dropdown of kwh/m2 or GJ/m2...